causallearn/utils/cit.py

```diff
@@ -59,9 +59,18 @@ def _unique(column):
         }
 
     def kci(self, X, Y, condition_set):
+        if condition_set == None:
+            condition_set = []
```

Comment:
This is not necessary. None condition_set is already handled here:
https://github.com/cmu-phil/causal-learn/blob/ffe75f95c4003fa7e9d7d5f3bbec4ace90ed3a41/causallearn/utils/cit.py#L337